Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:alpine
+WORKDIR /Rest-api
```

Comment:
just use lowercase everywhere possible in non-code scenario

Reply:
I've made changes to ensure lowercase everywhere in code.